warp/_src/localized.py

```
@@ -0,0 +1,1818 @@
+    if dtype in dtype_map:
+        cp_dtype = dtype_map[dtype]
+        np_dtype = cp.dtype(cp_dtype)
+        elem_size_bytes = np_dtype.itemsize
+    elif isinstance(dtype, str):
+        cp_dtype = dtype  # String dtype
+        np_dtype = np.dtype(dtype)
+        elem_size_bytes = np_dtype.itemsize
+    else:
+        # Structured type (vec2, vec3, mat22, etc.)
+        # CuPy can't handle Warp's structured types, so allocate as bytes
+        is_structured_type = True
```

Ruff (0.14.2)

Local variable `is_structured_type` is assigned to but never used

Remove assignment to unused variable `is_structured_type`

(F841)